Changes to reduce build time and size

n-sandeep opened the pull request:
- Because of debug option, size of image is increased. Removing build with -g option. User need to add this flag if needed for debugging.
- Disabling SSL when we build with default path as well.

Signed-off-by: Venkata Suresh Kumar P <venkata.suresh.kumar.p@intel.com>

ipdksw commented:
Result:FAILURE JenkinsConsole:https://s3.ipdk.io/ovs/PR-68-1/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/64a4aae53d1b10a74a139ca08d1de5fcddbe14a2

n-sandeep committed:
Changes to reduce build time and size

- Because of debug option, size of image is increased.
Removing build with -g option. User need to add this flag if needed for debugging.
- Disabling SSL when we build with default path as well.

Signed-off-by: Sandeep N <sandeep.nagapattinam@intel.com>

n-sandeep commented:
build failure is due to error in PTF script.
p4ovs_test_runner.py: error: argument -f/--file: expected one argument

nupuruttarwar reviewed (commented):
I see, gflags have following options:
BUILD_TESTING | Build tests for execution by CTest.
BUILD_NC_TESTS | Request inclusion of negative compilation tests (requires Python).
BUILD_CONFIG_TESTS | Request inclusion of package configuration tests (requires Python).

Should we turn these OFF too explicitly?

For glog, -DWITH_GTEST=OFF?

nupuruttarwar commented on build-p4ovs.sh:
```diff
@@ -41,9 +41,9 @@ fi
+    ./configure --prefix=$DEPS_INSTALL_PATH --with-p4tdi=$SDE_INSTALL_PATH CFLAGS='-O0' --disable-ssl --with-sai || exit 1
 else
-    ./configure --with-p4tdi=$SDE_INSTALL_PATH CFLAGS='-O0 -g' --with-sai || exit 1
+    ./configure --with-p4tdi=$SDE_INSTALL_PATH CFLAGS='-O0' --with-sai || exit 1
```
The commit message says disable-ssl for default path too, I think that option is missed on line 46

n-sandeep replied:
You are right., missed adding disable-ssl in else case. Will add in next patch

ipdksw commented:
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-68-2/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/64a4aae53d1b10a74a139ca08d1de5fcddbe14a2

ipdksw commented:
Result:FAILURE JenkinsConsole:https://s3.ipdk.io/ovs/PR-68-3/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/74016287a7545ea0ea881979697c006d46a1650f

vsureshkumarp reviewed (approved):
Changes looks good to me.

ipdksw commented:
Result:SUCCESS JenkinsConsole:https://s3.ipdk.io/ovs/PR-68-4/index.html GitCommit:https://github.com/ipdk-io/ovs/commits/be8d155c1b94c045da16a8d8acbc9db1b94778ff

nupuruttarwar reviewed (approved):
LGTM
